ansible/roles/texlive/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+texlive_dest: /opt/texlive
+texlive_version: 2015
+# The above is duplicated (!) in the installer profile file - ../files/
```

Comment:
Either get rid of the vars below and just use the full var definitions in the template, or prefix them with `texlive` (and lower-case to make them cxonsistent?)